### PR TITLE
Adding KlugDossier

### PR DESCRIPTION
**Pull request by sbnair**

### CODE REVIEW

**Review by sbnair — COMMENTED** [review]
Please review the new changes.

File: .github/workflows/deno.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This workflow uses actions that are not certified by GitHub.
```
Comment (jacogr) [inline .github/workflows/deno.yml:1]: Not sure this is needed at all? It is not something that the team can actually maintain going forward.

File: packages/apps-config/src/ui/colors.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ const chainStatemine = '#111';
 const chainWestend = '#da68a7';
 const chainGalois = '#000000';
 const chainZero = '#000000';
+const chainKlug = '#000000';
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:69]: Order these alphabetical.

File: packages/apps-config/src/ui/colors.ts [hunk 3]
```diff
@@ -124,6 +125,7 @@ const nodeWeb3games = 'linear-gradient(45deg, #0099F7 0%, #F11712 100%)';
 const nodeWestlake = chainWestlake;
 const nodeZeitgeist = 'linear-gradient(180deg, rgba(32,90,172,1) 0%, rgba(26,72,138,1) 50%, rgba(13,36,69,1) 100%)';
 const nodeZero = '#0099cc';
+const nodeKlug = '#663399';
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:128]: Order these alphabetical.

File: packages/apps-config/src/ui/colors.ts [hunk 4]
```diff
@@ -237,6 +239,7 @@ export const chainColors: Record<string, string> = [
   ['Westmint Test', nodeStatemint],
   ['Westlake', chainWestlake],
   ['Zenlink PC1', chainZenlink],
+  ['KLUGDOSSIER.NET', chainKlug],
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:242]: Order these alphabetical.

File: packages/apps-config/src/ui/colors.ts [hunk 5]
```diff
@@ -300,6 +303,7 @@ export const nodeColors: Record<string, string> = [
   ['Westmint Collator', nodeStatemint],
   ['Zeitgeist Node', nodeZeitgeist],
   ['Zeitgeist Collator', nodeZeitgeist],
+  ['Klug Dossier Node',nodeKlug]
```
Comment (jacogr) [inline packages/apps-config/src/ui/colors.ts:306]: Order these alphabetical.

File: packages/apps-config/src/ui/logos/index.ts [hunk 6]
```diff
@@ -105,6 +105,7 @@ import nodeWestlake from './nodes/westlake.png';
 import nodeZeitgeist from './nodes/zeitgeist.png';
 import nodeZenlink from './nodes/zenlink.svg';
 import nodeZero from './nodes/zero.svg';
+import nodeKlug from './nodes/klug.png';
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:108]: Order these alphabetical.

File: packages/apps-config/src/ui/logos/index.ts [hunk 7]
```diff
@@ -191,8 +192,10 @@ export const chainLogos: Record<string, unknown> = [
   ['Vln', nodeVln],
   ['Web3games', nodeWeb3games],
   ['Westlake', nodeWestlake],
+  ['KlugDossier',nodeKlug],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:195]: Order these alphabetical.

File: packages/apps-config/src/ui/logos/index.ts [hunk 8]
```diff
@@ -315,6 +318,7 @@ export const nodeLogos: Record<string, unknown> = [
   ['Zenlink', nodeZenlink],
   ['Zenlink Collator', nodeZenlink],
   ['Zeitgeist Node', nodeZeitgeist],
+  ['Klug Dossier Node',nodeKlug],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:321]: Order these alphabetical.

File: packages/apps-config/src/ui/logos/index.ts [hunk 9]
```diff
@@ -467,7 +471,8 @@ export const namedLogos: Record<string, unknown> = {
   westlake: nodeWestlake,
   westmint: nodeStatemine,
   zeitgeist: nodeZeitgeist,
-  zero: nodeZero
+  zero: nodeZero,
+  KlugDossier: nodeKlug
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:475]: Order these alphabetical.

**Review by jacogr — COMMENTED** [review]
The "Order these alphabetical" may sound like OCD, but the issue is there are a lot of these, so trying to maintain some sense of sanity :)

(CI will complain about some of them anyway, but doesn't catch all)

File: .github/workflows/deno.yml (hunk 1 above)
Comment (sbnair) [reply .github/workflows/deno.yml]: OK,I have removed this.

File: packages/apps-config/src/ui/colors.ts (hunk 2 above)
Comment (sbnair) [reply packages/apps-config/src/ui/colors.ts:69]: Ok, I ordered it.

File: packages/apps-config/src/ui/colors.ts (hunk 4 above)
Comment (sbnair) [reply packages/apps-config/src/ui/colors.ts]: OK, I have arranged it alphabatically.

File: packages/apps-config/src/ui/logos/index.ts (hunk 6 above)
Comment (sbnair) [reply packages/apps-config/src/ui/logos/index.ts]: OK, I arranged it now.

File: packages/apps-config/src/ui/logos/index.ts (hunk 8 above)
Comment (sbnair) [reply packages/apps-config/src/ui/logos/index.ts]: Ok, I arranged it now.

File: packages/apps-config/src/ui/logos/index.ts (hunk 9 above)
Comment (sbnair) [reply packages/apps-config/src/ui/logos/index.ts]: I arranged it in new commit.

**Review by sbnair — COMMENTED** [review]
Please review my changes and allow.

**Comment by jacogr** [comment]
CI should be auto-fixable by running `yarn lint` locally.

**Comment by sbnair** [comment]
> CI should be auto-fixable by running `yarn lint` locally.

I am using 3.0.0-rc.2 version of yarn and when i run 'yarn lint' i get this error : command not found: polkadot-dev-run-lint

**Comment by jacogr** [comment]
- clone
- `yarn` to install dependencies (first step, needed)
- `yarn start` to run it locally (for testing, connect to http://localhost:3000)
- `yarn lint` to auto-fix linting issues

**Pull request closed without merging**